s3buildcache/build.gradle.kts

```diff
@@ -51,7 +51,7 @@ gradlePlugin {
 }
 
 group = "androidx.build.gradle.s3buildcache"
-version = "1.0.0-beta01"
+version = "1.0.0-alpha01"
```

Comment:
This moved to an alpha as we've never published it before (s3 variant of the plugin)